tests/python/unittest/test_node_reflection.py

```diff
@@ -57,7 +57,7 @@ def test_make_smap():
 
 
 def test_make_node():
-    x = tvm.ir.make_node("IntImm", dtype="int32", value=10)
+    x = tvm.ir.make_node("IntImm", dtype="int32", value=10, span=None)
```

Comment:
do we have to pass in span? or can it be optionally passed in here

Reply:
Right now we have to pass in the span. The problem is that the ffi doesn't really allow for keyword arguments. We can fake them by using optional arguments, but this fails when there are multiple optional arguments. I opted for making things consistent, so I don't allow optional spans at the ffi barrier. Instead, the optional spans are handled by the python wrapper functions. Unfortunately, this approach means that make_node needs to explicitly pass the span.

Reply:
OK sounds good, in most cases we are using constructor instead of make_node so hopefully it won't be a problem